infra/modules/providers/azure/app-gateway/output.tf

```diff
@@ -12,3 +12,11 @@ output "appgateway_frontend_ip_configuration" {
   description = "The Application Gateway Frontend IP Configuration"
   value       = "${azurerm_application_gateway.appgateway.frontend_ip_configuration}"
 }
+
+output "appgateway_health_probe_backend_status" {
+  value = "${lookup(data.external.app_gw_health.result, "health")}"
```

Comment:
will this eventually be a list once we support multiple backends? Can we model as such now?

Similar comment r.e. `app_gateway_health_probe_backend_address`

Reply:
> Couple things to start:
> 
> 1. Need direction for changing my TF state to have ianphil appended to it so we don't step on each other.
> 2. Need direction on where in the files to I change the pre/postfix so that we don't have name collisions again.
> 3. I don't see anything refrencing the `.env` file or the `dotenv` function

Thanks for the comment @iphilpot . 

I added a [mention](https://github.com/microsoft/cobalt/tree/erisch/templates/azure-simple/feature/azkeyvault/infra/templates/azure-simple#example-usage) in the template readme on how to create a TF workspace. I append the workspace name to the deployed resource names which addresses item 2. The workspace name is set to the `$USERNAME` environment variable which is defined by default on all our desktops.

Reply:
Unresolved as the comment is to a different question, but don't block on this.